scripts/style-checker.sh

```diff
@@ -52,12 +52,35 @@ function astyle_c_format
     exit 0
 }
 
+function astyle_c_diff
+{
+    astyle --options="$root_dir/.astylerc" "$root_dir/*.c" |
+    grep "Formatted" |
```

Comment:
I don't understand the question. Which lines? Enough for what? Calling it when after what?